tasks/gen/action.cr

```diff
@@ -0,0 +1,53 @@
+require "colorize"
+require "ecr"
+require "file_utils"
+
+class Lucky::ActionGenerator
+  getter :name
+
+  ECR.def_to_s "#{__DIR__}/action.ecr"
+
+  def initialize(@name : String)
+  end
+
+  def generate
+    make_folders_if_missing
```

Comment:
I love this method name!

Reply:
I stole it from your migration generator 😛

Reply:
Lol :P